src/RedHat.AspNetCore.Server.Kestrel.Transport.Linux/TransportThread.ThreadContext.cs

```diff
@@ -779,6 +818,8 @@ private void TrySendToPipe(byte operation)
 
             public unsafe void Dispose()
             {
+                AbortQueuedConnectionAsync().GetAwaiter().GetResult();
```

Comment:
Would be better if we use this?

```suggestion
               Task.WaitAll(AbortQueuedConnectionAsync());
```

Reply:
This would wrap the exception in an `AggregateException`.